### PR TITLE
Added documentation about Tella FOSS

### PR DESCRIPTION
**Pull request by carohadad**

### CODE REVIEW

File: docs/privacy-policy.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Tella also collects information about your mobile device and internet connection
 
 Tella also collects information that helps identify the location of the device when a file was captured. This includes location (including latitude, longitude, accuracy, and altitude), location provider, location speed, and cell towers that your device is connected to. Collectively, this data is known as “Location Metadata”.
 
-File, device and location metadata are only collected if you opt-in by turning on Verification Mode in your App settings. If you do not want Tella to collect this information, do not activate Verification Mode.
+File, device and location metadata are only collected if you opt-in by turning on Verification Mode in your App settings.  If you do not want Tella to collect this information, do not activate Verification Mode.
```
Comment (djordjemarusic) [inline docs/privacy-policy.md]: Unnecessary space was added.

Comment (carohadad) [reply docs/privacy-policy.md]: thanks!!

**Review by djordjemarusic — APPROVED** [review]

**Comment by carohadad** [comment]
please djordje and raph: take a look at the release notes and check if you want to add more details about what changed for the FOSS version

**Review by djordjemarusic — APPROVED** [review]

File: docs/Discover Tella/whats-new.md [hunk 2]
```diff
@@ -26,3 +26,7 @@ Launched Reports, a quick way to back-up media files to your organization's serv
 Also in this update:
 - Delete the original file from the phone's file system after importing it into Tella.
 - Configure a quick delete button to erase the content of Tella in an emergency.
+
+## Tella FOSS 2.0.15 (based on Android 2.0.15) - July 10, 2023
+
+A version of Tella included for the first time on the F-droid store. This a 100% Free and Open-Source Software (FOSS) version of Tella Android. We removed all trackers, changed map and location provider and also changed the Camera library to CameraX, removed crashlytics, LoggingInterceptor and any other non-FOSS component or dependency.
```
Comment (djordjemarusic) [inline docs/Discover Tella/whats-new.md]: We should add that we removed Google Play services too.

**Comment by raphmim** [comment]
I made a few fixes and changes here: https://github.com/Horizontal-org/Tella-Docs/pull/11/commits/415a13c92b1a2833d8689a49d2d3213b2ec10193

The only big(ish) changes are:
- I tried to streamline the language around Tella-FOSS, so that it's the same across the documentation, and linking to the FAQ about it
- I removed some of the in-depth language from the Get Started sections to keep them short and simple, given that those are meant for end users and not project managers or admins.

**Pull request merged**